src/Core/TileGeometry.js

```diff
@@ -86,7 +86,7 @@ TileGeometry.prototype.computeBuffers = function computeBuffers(params, builder)
         outBuffers.index = cachedBuffers.index;
         outBuffers.uv.wgs84 = cachedBuffers.uv.wgs84;
     } else {
-        scratchBuffers.index = new Uint32Array(triangles * 3 * 2);
```

Comment:
there was too much index in all case?

Reply:
No I move the `* 2` in the triangle count computation.
The indices count was correct but is better expressed as `num_triangles * 3`

Reply:
ok